### PR TITLE
halit_week1_node

### PR DESCRIPTION
**Pull request by halitos**
Node.JS week 1 mandatory HW complete

### CODE REVIEW

File: week-1/mandatory/2-dev-tools/task.md [hunk 1]
```diff
@@ -7,12 +7,30 @@ In Google Chrome, open dev tools network tab and enter this URL into the browser
 Answer the following questions:
 
 - What's the status code the server sent back?
+
+  - Status Code: 304 Not Modified and 200 for the image(png)
+
```
Comment (Ashaghel) [inline week-1/mandatory/2-dev-tools/task.md:12]: when you will get 304? how can you get 200 
Search clearing cached

File: week-1/mandatory/2-dev-tools/task.md [hunk 2]
```diff
@@ -7,12 +7,30 @@ In Google Chrome, open dev tools network tab and enter this URL into the browser
...
+  - Request Method : GET
+
 - What is the request path?
+
+  - /shows/82/episodes
+
 - What is the first line of the response body?
+
+  - beginning of the array of objects...
```
Comment (Ashaghel) [inline week-1/mandatory/2-dev-tools/task.md:23]: The response is back with one line only. The whole array will come back, check it in the dev tools

File: week-1/mandatory/2-dev-tools/task.md [hunk 3]
```diff
@@ -7,12 +7,30 @@ In Google Chrome, open dev tools network tab and enter this URL into the browser
...
+
 - What is the value of the response header called "Content-Type" ?
+
+  - Content-Type: application/json
+
 - What is the value of the request header called "User-Agent"
 
+  - User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko)
+    Chrome/84.0.4147.105 Safari/537.36
+
```
Comment (Ashaghel) [inline week-1/mandatory/2-dev-tools/task.md:33]: Great work

**Comment by halitos** [comment]
Thank you for review, Ahmad. Closing this PR now to open new one for week 2.

**Pull request closed without merging**